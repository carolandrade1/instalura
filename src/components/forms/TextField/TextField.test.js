import React from 'react';
import { render, screen } from '@testing-library/react';

import TextField from './index';
import WebSiteGlobalProvider from '../../wrappers/WebsitePage/provider';

describe('<TextField />', () => {
  test('renders component', () => {
    render(
      <WebSiteGlobalProvider>
        <TextField placeholder="Nome" value="ju" onChange={() => { }} name="nome" />
      </WebSiteGlobalProvider>,
    );

    const textField = screen.getByPlaceholderText(/nome/i);
    expect(textField).toMatchSnapshot();
  });
});
