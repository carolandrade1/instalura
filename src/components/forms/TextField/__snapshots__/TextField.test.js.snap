// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextField /> renders component 1`] = `
<input
  class="Text__TextBase-sc-1m3ih9v-0 iWpvrZ TextField__Input-sc-15t91h4-1 JnDIw"
  href=""
  name="nome"
  placeholder="Nome"
  type="text"
  value="ju"
/>
`;
